libbpf-rs/src/skeleton.rs

```diff
@@ -165,10 +165,10 @@ impl<'a> ObjectSkeletonConfigBuilder<'a> {
         // Holds `CString`s alive so pointers to them stay valid
         let mut string_pool = Vec::new();
 
-        // NB: use default() to zero out struct
```

Comment:
@danobi I also got rid of this, since it appears to be able zeroing out the bytes, but it sounds like we'll fix it with the bind-gen bugfix